ua_parser/user_agent_parser.py

```diff
@@ -234,14 +236,14 @@ def ParseUserAgent(user_agent_string, **jsParseBits):
     Returns:
       A dictionary containing parsed bits.
     """
-    if 'js_user_agent_family' in jsParseBits and jsParseBits['js_user_agent_family'] != '':
+    if (
+        'js_user_agent_family' in jsParseBits and
+        jsParseBits['js_user_agent_family'] != ''
+    ):
         family = jsParseBits['js_user_agent_family']
-        if 'js_user_agent_v1' in jsParseBits:
-            v1 = jsParseBits['js_user_agent_v1'] or None
-        if 'js_user_agent_v2' in jsParseBits:
-            v2 = jsParseBits['js_user_agent_v2'] or None
-        if 'js_user_agent_v3' in jsParseBits:
-            v3 = jsParseBits['js_user_agent_v3'] or None
+        v1 = jsParseBits.get('js_user_agent_v1')
```

Comment:
These also aren’t equivalent code. So I’m not sure what the goal here is or if the change in behavior is fine.

Reply:
aren't they? may i know in which case? I think they are same

Reply:
```python
>>> jsParseBits = {'js_user_agent_v1': False, 'js_user_agent_v2': ''}
>>> jsParseBits.get('js_user_agent_v1')
False
>>> jsParseBits['js_user_agent_v1'] or None
>>>
```

If the key exists, but it's any falsey value. The falsey value will be returned instead of None.

In theory, `v1 = jsParseBits.get('js_user_agent_v1') or None` would be almost equivalent and shortened.

The only difference here now is that `v1` is being defined guaranteed, whereas before it wasn't. Though I'd agree having it guaranteed defined would be a bit better to avoid any potential NameError.

Reply:
oh, thanks. fix in https://github.com/ua-parser/uap-python/pull/80/commits/9da6ff60314a33bc72b7905498ee987d2c17577b